src/components/dialogs/generator-creation-dialog.js

```diff
@@ -238,14 +255,13 @@ const GeneratorCreationDialog = ({
     });
 
     const [reactiveCapabilityCurve, reactiveCapabilityCurveField] =
-        useTableValues({
-            id: 'ReactiveCapabilityCurveOn',
+        useReactiveCapabilityCurveTableValues({
             tableHeadersIds: headerIds,
             inputForm: inputForm,
             Field: ReactiveCapabilityCurveTable,
             defaultValues: formValues?.reactiveCapabilityCurvePoints,
             isRequired: false,
```

Comment:
`isRequired` seems not used anymore in this custom hook ?

Reply:
Correct.